server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/vault/RabbitMQDeletedMessageVaultIntegrationTest.java

```diff
@@ -42,7 +42,7 @@
     public GuiceJamesServer createJmapServer(FileSystem fileSystem, Clock clock) throws IOException {
         return rule.jmapServer(cassandra.getModule(),
```

Comment:
`DeletedMessageVaultIntegrationTest` is a special case, I will do it in a separated PR

Reply:
not any more